tests/ui/async-await/issue-64130-3-other.stderr

```diff
@@ -2,7 +2,7 @@ error[E0277]: the trait bound `Foo: Qux` is not satisfied in `impl Future<Output
   --> $DIR/issue-64130-3-other.rs:25:12
    |
 LL | async fn bar() {
-   |                - within this `impl Future<Output = ()>`
+   |               - within this `impl Future<Output = ()>`
```

Comment:
This is the only weirdness. I want to open a separate PR to actually just point to the `async` keyword span, but I don't think there's actually any "natural" place to point here to mean "the "`impl Future` that comes from this desugaring"...

Reply:
When you make that change, would pointing to the whole function signature (through the return type if it exists) give more context as to where the `impl Future` comes from? A rewording might help too, though I'm not sure how easy that is to do.

something like
```
async fn meow() -> UwU {
^^^^^^^^^^^^^^^^^^^^^^ within the `impl Future<Output = UwU>` created by this `async` fn
```